registrations/tasks.py

```diff
@@ -287,3 +290,23 @@ def run(self, registration_id, **kwargs):
         return validation_string
 
 validate_registration = ValidateRegistration()
+
+
+class DeliverHook(Task):
```

Comment:
Please refresh this from the source at https://github.com/praekelt/hellomama-registration/blob/develop/registrations/tasks.py#L265-L290